scripts/pipelines/github/templates/build/build-pipeline.yml.template

```
@@ -0,0 +1,38 @@
+name: $pipelineName
+
+on:
+  workflow_call:
+  workflow_dispatch:
+    inputs:
+      target: 
```

```suggestion
      targetPath: 
```
To be consistent with the azure pipeline